EmployeeManagement/Tavisca.EmployeeManagement.Translator/Translator.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.Serialization;
+using System.ServiceModel;
+using System.Text;
+using Tavisca.EmployeeManagement.DataContract;
+using Tavisca.WCF.Model;
+                         
+namespace Tavisca.EmployeeManagement.Translator
+{
+    // NOTE: You can use the "Rename" command on the "Refactor" menu to change the class name "Service1" in both code and config file together.
+    public static class Translator 
+    {
+        public static EmployeeModel EmployeeModelGenerator(Employee employee)
+        {
+
+            var employeeModel = EmployeeModelFactory.CreateInstance(employee.Title,employee.FirstName, employee.LastName, employee.Email);
```

Comment:
What is this? Why do I need a factory here? business model employee is just a regular class, why do you need to create an instance using a factory?